Formula/lmdb.rb

```diff
@@ -16,20 +15,15 @@ class Lmdb < Formula
   end
 
   def install
-    inreplace "libraries/liblmdb/Makefile" do |s|
-      s.gsub! ".so", ".dylib"
-      s.gsub! "$(DESTDIR)$(prefix)/man/man1", "$(DESTDIR)$(prefix)/share/man/man1"
+    cd "libraries/liblmdb" do
+      inreplace "Makefile", ".so", ".dylib"
```

Comment:
Would be good to get this reported upstream. Switching extensions around based on platforms is a pretty easy thing to support.